rbac/management/role/view.py

```diff
@@ -492,6 +492,12 @@ def perform_update(self, serializer):
 
         Assumes concurrent updates are prevented (e.g. with atomic block and locks).
         """
+        if serializer.instance.system:
```

Comment:
is it possible to move this in validation method on serializer ? I mean Object-Level Validation - https://www.django-rest-framework.org/api-guide/serializers/#object-level-validation 

it should be called before replication.

Reply:
Yea, good idea updated